src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -0,0 +1,99 @@
+.profile-container {
+  font-size: 0.9em;
+  width: 80%;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 100px;
+  margin-left: auto;
+
+  button {
+    margin: 30px;
+  }
+
+  .header {
```

Comment:
we already have header, let it be ubs_profile-header

Reply:
done